tests/test_string_spaced_pretty.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
```

Comment:
There's no need to create a whole new set of code for this.  test1Formatted and test2Formatted will both happily take multiple flags, though you'll need to tweak test2.test to do something like

    run_output_test -o test2Formatted_spaced_pretty test2Formatted spaced pretty

since the existing loop won't handle that.